parser/parser.y

```diff
@@ -3437,6 +3437,11 @@ Statement:
 |	UnionStmt
 |	UpdateStmt
 |	UseStmt
+|	SubSelect
```

Comment:
I think here may be `'(' SelectStmt ')', not SubSelect, in MySQL, SubSelect is not the same as select. 
And you can see the MySQL parser that it uses select too, not sub select.